libc/test/src/math/RIntTest.h

```diff
@@ -131,7 +138,7 @@ class RIntTestTemplate : public LIBC_NAMESPACE::testing::FEnvSafeTest {
       for (int mode : ROUNDING_MODES) {
         LIBC_NAMESPACE::fputil::set_round(mode);
         mpfr::RoundingMode mpfr_mode = to_mpfr_rounding_mode(mode);
-        ASSERT_FP_EQ(func(x), mpfr::round(x, mpfr_mode));
+        EXPECT_FP_EQ(func(x), mpfr::round(x, mpfr_mode));
```

Comment:
Would it be worth it to also use `EXPECT_FP_EQ_WITH_EXCEPTION` here based on the mantissa of `x`/`xbits`?